std/http/file_server.ts

```diff
@@ -353,7 +353,11 @@ function main(): void {
           setCORS(response);
         }
         serverLog(req, response!);
-        req.respond(response!);
+        try {
+          await req.respond(response!);
```

Comment:
There shouldn't be an await here. We want to asynchronously respond to requests so others can be handled.

Reply:
This handler function is not awaited for in `server.ts` `listenAndServe`: https://github.com/denoland/deno/blob/master/std/http/server.ts#L279

So I suppose async requests still works. I'm just using await because otherwise the handler function will end before the error is thrown (and will thus not be caught).

Is that correct? Otherwise I'll change it to use `catch()` on the respond promise.